staging/src/k8s.io/client-go/tools/clientcmd/api/v1/zz_generated.conversion.go

```diff
@@ -401,6 +401,7 @@ func autoConvert_api_ExecConfig_To_v1_ExecConfig(in *api.ExecConfig, out *ExecCo
 	out.InteractiveMode = ExecInteractiveMode(in.InteractiveMode)
 	// INFO: in.StdinUnavailable opted out of conversion generation
 	// INFO: in.StdinUnavailableMessage opted out of conversion generation
+	// INFO: in.PluginPolicy opted out of conversion generation
```

Comment:
What is the reason for this opt-out? Is it to ensure we're not exposing these fields in the v1 part of this API?